app/components/article-comments.js

```diff
@@ -68,6 +68,14 @@ export default Component.extend(InViewportMixin, {
 
     this._super(...arguments);
 
+    const { urlThreadId } = this.articleComments.getUrlThreadParams();
+
+    if (this.isUcp && urlThreadId) {
+      this.articleComments.fetchCount(this.articleTitle, this.articleNamespace)
```

Comment:
it's already called in `didEnterViewport`, so it seems it'll be called twice

Reply:
I will fetch this in `didInsertElement` then